Source/WebKit/Shared/WebPushMessage.h

```diff
@@ -25,13 +25,11 @@
 
 #pragma once
 
+#include <WebCore/NotificationPayload.h>
```

Comment:
Could `WebCore::NotificationPayload` be forward declared?

Reply:
Of all the "This could be forward declared?" comments this one hurts the most in the number of files I'd have to go touch.

The frustration here is that I can get it building locally, but then the wildly different configs on the bots (iOS, tv, older Mac, linuxes) all have different notions of unified sources.

I tried to resolve these forward declare vs includes multiple times today, but can't afford to spend any more time back-and-forthing with EWS on this =(

Reply:
If it's a huge issue, don't bother. I've just trained myself to watch for these includes since it's a lot of work to remove them later (as you know!) :-)